### DISCUSSION TITLE
Realidade aumentada com AR.js e o suporte ao iOS

### DISCUSSION THREAD

**Discussion by fxcosta** [frontendbr/forum#784] [Perguntas e Respostas]
Olá, pessoal!

Comecei a brincar com realidade aumentada usando o [AR.js](https://github.com/jeromeetienne/AR.js) mas pelo que percebi não há um suporte bom pra iOS por conta do WebRTC. Pedi pra um colega testar e realmente ele não conseguiu rodar os exemplos.

A minha dúvida é se há ou se dá pra fazer algum polyfill do WebRTC pra iOS e fazer funcionar bem realidade aumentada usando o poder da web

Alguém mais trabalhou com realidade aumenta ou virtual usando somente a web e teve que lidar com esses problemas com suporte? Obrigado!!

**Comment by felipeuntill** [comment]
O IOS 11 liberou o suporte para WebRTC nativo, se o device estiver atualizado irá funcionar.

[iOS 11 and WebRTC Support](https://www.quobis.com/2017/06/08/webrtc-ios11/)
